### PR TITLE
Apply yapf

### PR DESCRIPTION
**Pull request by hirosassa**
SSIA

### CODE REVIEW

File: yapf.ini [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [style]
 based_on_style = pep8
-column_limit = 120
+column_limit = 160
```
Comment (Hi-king) [inline yapf.ini:3]: You mean, we've got wider display now? :)

Comment (hirosassa) [reply yapf.ini:3]: Yeah, my display width is ... kidding.

just following gokart and thunderbolt
https://github.com/m3dev/gokart/blob/master/yapf.ini
https://github.com/m3dev/thunderbolt/blob/master/yapf.ini

**Review by Hi-king — APPROVED** [review]
There seem to be no difference in code. Looks OK 🙆‍♀️

File: yapf.ini (hunk 1 above)
Comment (Hi-king) [reply yapf.ini:3]: 👍

**Pull request merged**